src/main/src/test/java/org/geoserver/data/test/MockData.java

```diff
@@ -56,6 +57,8 @@
  * @author Justin Deoliveira, The Open Planning Project
  *
  */
+
```

Comment:
@NielsCharlier there should be no blank lines between the class Javadoc and the class definition. Other than this, your changes to gs-main and the gs-web-app pom look fine to me.